### ISSUE TITLE
Android error on flatlist item that has Touchable

### ISSUE THREAD

**Issue by plrdev** [react-native-community/react-native-platform-touchable#3]
I have a flatlist with an item like this:

      <Touchable onPress={() => this.props.onPress()} background={Touchable.SelectableBackground()}>

It usually works, but sometimes I get this error:

Attempted to transition from state `RESPONDER_INACTIVE_PRESS_IN` to `RESPONDER_ACTIVE_LONG_PRESS_IN`, which is not supported. This is most likely due to `Touchable.longPressDelayTimeout` not being cancelled.

**Comment by HZSamir** [comment]
@eduplus I'm guessing you're debugging your app over Chrome Debugger, as this is a bug that appears often with it
Nothing you can do about it. Either reload your app each time this happens, or do debug JS remotely
I agree it's bit vexing

**Comment by plrdev** [comment]
Yeah debugging with Chrome. If it only happens during debugging, then that's fine I guess.
